packages/web-react/src/components/Pagination/stories/UncontrolledPagination.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { UncontrolledPagination } from '..';
+
+const meta: Meta<typeof UncontrolledPagination> = {
+  title: 'Components/Pagination',
+  component: UncontrolledPagination,
+  parameters: {
+    layout: 'centered',
+  },
+  argTypes: {
+    defaultPage: {
+      control: 'number',
+      description: 'Default page for the first render; please reload the page to apply this setting.',
+      table: {
+        defaultValue: { summary: 1 },
+      },
+    },
+    totalPages: {
+      control: 'number',
+      table: {
```

Comment:
you can remove this. There is no default and you set the prop in the `args` section.